### PR TITLE
refactor: alignments with Flake8, Black and Pydocstyle linters

### PR DESCRIPTION
**Pull request by robertstypa**
Refactor: alignments with Flake8, Black and Pydocstyle linters as a preparation to merge `ncs` into the `main` branch

Ref: None

Signed-off-by: Robert Stypa <robert.stypa@nordicsemi.no>

### CODE REVIEW

File: suit_generator/cmd_cache_create.py [hunk 1]
```diff
@@ -13,23 +13,37 @@
...
 
 
 def add_padding(data: bytes, eb_size: int) -> bytes:
-    data_len = len(data)
-
+    """
+    Add padding to the given data to align it to the specified erase block size.
+
+    This function ensures that the data is padded to a size that is a multiple of the erase block size (eb_size).
+    The padding is done according to the CBOR (Concise Binary Object Representation) format.
```
Comment (tomchy) [inline suit_generator/cmd_cache_create.py:39]: You may also add a more detailed explanation here as this function does not append a dumb, constant padding:

The padding is done by appending a CBOR  key-value pair with empty URI as the key and byte-string-encoded zeros as the value.

File: tests/test_cmd_cache_create.py [hunk 2]
```diff
@@ -15,43 +15,205 @@
...
-                             0x5A, 0x00, 0x00, 0x00, 0x04, # bstr size 4
-                             0x01, 0x02, 0x03, 0x04, # BINARY_CONTENT_1
-                             0x60, 0x45, 0x00, 0x00, 0x00, 0x00, 0x00, # padding
-                             0x67, 0x23, 0x73, 0x65, 0x63, 0x6F, 0x6E, 0x64, # tstr "second"
-                             0x5A, 0x00, 0x00, 0x00, 0x0a, # bstr size 10
-                             0x08, 0x09, 0x0A, 0x0B, 0x0C, 0x0D, 0x0E, 0x0F, 0x10, 0x11, # BINARY_CONTENT_2
-                             0x60, 0x47, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, # padding
-                             0xFF
-                             ])
+EXPECTED_CACHE_EB_8 = bytes(
```
Comment (tomchy) [inline tests/test_cmd_cache_create.py:21]: Personally - preferred the previous formatting. Can we force black to ignore this section?

**Review by tomchy — APPROVED** [review]

File: tests/test_cmd_cache_create.py (hunk 2 above)
Comment (kszromek-nordic) [reply tests/test_cmd_cache_create.py:21]: I agree, following solution can be used:
https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#ignoring-sections

**Review by jnsgsr — APPROVED** [review]

File: tests/test_cmd_cache_create.py (hunk 2 above)
Comment (robertstypa) [reply tests/test_cmd_cache_create.py:21]: I agree, new formatting is very ugly - previous version will be restored.

**Review by kszromek-nordic — APPROVED** [review]

**Pull request merged**